src/main/java/fr/insee/lunatic/conversion/JSONDeserializer.java

```diff
@@ -17,42 +15,26 @@ public JSONDeserializer() {	}
 
 	private static final Logger logger = LoggerFactory.getLogger(JSONDeserializer.class);
 
-	public Questionnaire deserialize(String fileName) throws JAXBException {
+	public Questionnaire deserialize(String fileName) throws IOException {
```

Comment:
We might want to remove this method later on, or create methods using proper URL/URI/Path objects

Reply:
postpone this to a later date :)